examples/cpp/pyperf/PyPerf.cc

```diff
@@ -143,6 +143,16 @@ int main(int argc, char** argv) {
     return 1;
   }
 
+  // checking against 0 which technically is a legit offset, but it won't ever happen
+  // for these fields, so it's okay as a sentinel.
+  if (fsOffset == 0) {
+    std::fprintf(stderr, "--fs-offset must be given!\n");
+    return 1;
+  } else if (stackOffset == 0) {
+    std::fprintf(stderr, "--stack-offset must be given!\n");
+    return 1;
+  }
+
```

Comment:
Maybe mention how to get them?

Reply:
As in, using the repo of `bpf_get_fs_offset`?

Reply:
Yes

Reply:
Or maybe refer to gProfiler, but I think referring to your repo directly is better

Reply:
K will add